sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -37,18 +37,44 @@ This exporter sends traces to the configured Azure Monitor Resource using HTTPS.
 
 Refer to [`Program.cs`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo/Program.cs) for a complete demo.
 
+### Traces
+
 ```csharp
 using Azure.Monitor.OpenTelemetry.Exporter;
 using OpenTelemetry.Trace;
 
 OpenTelemetry.Sdk.CreateTracerProviderBuilder()
-    .AddSource("Demo.DemoClient")
-    .AddAzureMonitorTraceExporter(o => {
-        o.ConnectionString = "<Your Connection String>";
-    })
+    .AddAzureMonitorTraceExporter(o => o.ConnectionString = "<Your Connection String>")
```

Comment:
I think we should keep AddSource in here. Probably add this statement to make it clear where the source comes from.

`ActivitySource source = new ActivitySource("MyCompany.MyProduct.MyLibrary");`

Reply:
I want to discuss this.

On the Readme doc, I think we should document either 1) the bare minimum to add our Exporter to a Provider or 2) a fully encapsulated example which includes the static ActivitySource or Meter.

I think the full example is covered by the Demo projects.
For our Readme, since this repo only owns the exporter I would argue that it's fine to have that minimal example showing simply the AddAzureMonitorExporter().